validateOffload.py

```diff
@@ -81,6 +81,12 @@ def run_ethtool_cmd(self, ethtool_cmd: str) -> Result:
         return r
 
     def parse_packets(self, output: str, packet_type: str) -> int:
+        prefix = f"{packet_type}_packets"
+        if prefix in output:
+            for line in output.splitlines():
+                stripped_line = line.strip()
+                if stripped_line.startswith(prefix):
+                    return int(stripped_line.split(":")[1])
```

Comment:
Smart change, my bad removing that.
The function is definitely more complicated now, it could make sense to refactor it.
Or maybe just add a comment like "case 1: rx_packets is in the ethtool output" "case 2: manually summing the queues"

Reply:
Done